### DISCUSSION TITLE
时局图

### DISCUSSION THREAD

**Discussion by ababcabcd** [WeNeedHome/SummaryOfLoanSuspension#347] [General]
![FXlqJlkUEAEcSwb](https://user-images.githubusercontent.com/77263247/178936509-3342dbe3-03a7-4f25-abe0-44a2c5b735be.jpeg)

**Comment by ruo-xi** [comment]
太真实了，我看b站 某些所谓的正能量的视频里面，发言都能把我气笑了，一堆人在哪里自嗨，不知道有几个做实事的？已有什么重大突破，一堆人在哪里婀娜奉承，他们又没什么贡献，我就不懂了 有什么好骄傲了。一说点国内的弊病，一定会有人说美国怎么样怎么样，摆脱我们至少宪法还是特色社会主义为什么要跟资本主义社会比烂，而且某些方面也属实不如西方做得好。

**Reply by Accelerator6666** [reply]
那些账号很多都是机器人。。。

**Reply by davidliu1645** [reply]
> 

那个火星方阵什么什么的就是机器人，关键是还好多粉丝

**Reply by summerBreeze03** [reply]
机器人罢了，我发句就被删

**Comment by luuthink** [comment]
典

**Comment by ethanrao** [comment]
4月份因为这张图 微信被封了一周 

转发到群里没人看得到，我以为被屏蔽了md5 checksum, 就截图又发了一遍，然后发出去半个小时，号就被封了，只能私聊；群聊和朋友圈用不了。

**Reply by dream80** [reply]
所以这张图里少画了至少两类人：往嘴里塞抹布禁言的。 还少了一个举大旗喊加速的。 （留言个言，感觉冒着生命危险）

**Reply by macqueen09** [reply]
机器审核+人工审核，图片加黑库了，类似以图搜图，所以加入黑库后 再截图发 也能匹配。除非   加特别多遮挡那种，变个色啥的有可能能绕开

**Reply by maxwillma** [reply]
> 机器审核+人工审核，图片加黑库了，类似以图搜图，所以加入黑库后 再截图发 也能匹配。除非 加特别多遮挡那种，变个色啥的有可能能绕开

惯犯啊！ ：）

**Comment by wedxz** [comment]
这个图没有喷子喷吗

**Reply by Accelerator6666** [reply]
太过于写实，无从开喷。

**Comment by GitHeP** [comment]
真实

**Comment by ppinkiris** [comment]
喷烂尾楼 的 和图上的观点并不冲突
我既喷烂尾楼的政府垃圾 同时觉得 其他船有2个洞你怎么不说
作图人为什么非要阉割掉这两种观点并行的可能呢？
同样的图换美国素材我也能做10张 都是片面看问题 高贵什么？
